### PR TITLE
debugging nautical.render.xml seabed icon and letter display

### PR DESCRIPTION
**Pull request by josail**
debugging the display of seabedDetail with regard to default property setting, letters and visibility of seagrass and weed icons

line icons seamark_j21_rocky and seamark_j22_coral and their corresponding night shaders rotated by 90 degree clockwise for correct display in OSMAND in line with INT-1 J21 and J22

### CODE REVIEW

**Comment by josail** [comment]
OSMAND_nautical_render_xml_1_2_14_2022_05_28
adjustments to nautical.render.xml are tested on iOS version of OSMAND as far as possible. 

The pathIcons for seamark_j21_rocky and seamark_j22_coral are rotated 90 degree clockwise but I did not find a way to upload the icons to the iOS OSMAND app and thereby test the effectivness => to be verified

**Comment by josail** [comment]
Question: Is there a method to apply the night shaders to the pathIcons? All night shader files are uploaded and available. If someone can apply them, this would help to highlight potentially dangerous rock or coral edges at reefs in night mode or for display in front of dark water in a background satellite image. Thanks if you know how to code and add this in the nautical.render.xml or any additional help/instructionn on this detail.

File: rendering_styles/nautical.render.xml [hunk 1]
```diff
@@ -1564,33 +1564,47 @@
...
+						<case tag="seamark:type" value="seabed_area"/> 
+						<case tag="seamark:type" value="weed"/>
+						<case tag="seamark:type" value="seagrass"/>
+					</switch>						
+			
+					<!-- omit seabedDetail of seamark -->
+					<switch seabedDetail="omit">
+						<case tag="seamark:type" maxzoom="18" value="seabed_area" nameTag="$null" nameTag2="$null"/> 
+						<case tag="seamark:type" minzoom="19" value="seabed_area" nameTag="seabed_surface_1"/><!-- in case SeabedDetail="omit" text is omitted, but a simplified seabed letter display at very high zoom level is maintained-->	
+						<case tag="seamark:type" value="weed" nameTag="$null" nameTag2="$null"/>
```
Comment (xmd5a2) [inline rendering_styles/nautical.render.xml:1604]: Please don't use $null as nameTag. It means color #00000000.

File: rendering_styles/nautical.render.xml [hunk 2]
```diff
@@ -3077,11 +3091,14 @@
 			</case>
    												
 			<!-- INT-1 J Section Nature of the Seabed -->
-			<!-- INT-1 J13, J14, J15 icons at single nodes-->
+			<!-- INT-1 J13, J14, J15 icons at single nodes -->
  			<switch minzoom="14" iconVisibleSize="1"> 
-  				<!-- weed and seagrass icons are renderd with display of the text, in order to avoid conflict with other important close by icons like rocks
-				<case tag="seamark:type" value="weed" icon="seamark_j132_weedkelp" iconOrder="5"/> 
-				<case tag="seamark:type" value="seagrass" icon="seamark_j134_seagrass" iconOrder="5"/> -->
+				<case tag="seamark:type" value="weed" icon="seamark_j132_weedkelp" iconOrder="1">
```
Comment (xmd5a2) [inline rendering_styles/nautical.render.xml:3096]: Why so high iconOrder?

**Pull request merged**